pylmod/gradebook.py

```diff
@@ -948,3 +1024,152 @@ def spreadsheet2gradebook(
             csv_reader, email_field, non_assignment_fields
         )
         return response
+
+    def get_staff(self, gradebook_id, simple=False):
+        """Get staff list for gradebook.
+
+        Get staff list for the gradebook specified.
+
+        If simple=True, return a list of dictionaries, one dictionary
```

Comment:
I misspoke.  I'm currently returning just displayName, accountEmail, and role for the "simple" list.